drivers/filesystems/nfs/nfs41_driver.c

```diff
@@ -19,6 +19,12 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA
  */
 
+/* NFSv4.1 relies on some NT5- functions */
```

Comment:
That would deserve a true explanation. I had to backport it to NT5, as it's initially written for Windows 7. So...

Reply:
Updated.